styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,3 +1,92 @@
+// Customer Stories Feature Colors
+$default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/IMACbanners/collaboration-2.jpg';
+
+.story-featured-container {
+  background-color: $gray-lightest;
+  padding: 40px 25px;
+}
+
+.story-featured-text {
+  @include make-container;
+  text-align: center;
+}
+
+.story-featured-cards {
+  @include make-container;
+}
+
+.story-freatured-cardRow {
+  @include make-row;
+  margin-top: 10px;
+}
+
+.story-featured-cardLogo {
+  @include make-md-column(3);
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.story-featured-logoInner {
+  @include flex-center;
+  box-shadow: 0 1px 3px transparentize($black, .5);
+  display: flex;
+  height: 220px;
+}
+
+.story-featured-cardQuote {
+  display: none;
+}
+
+.story-featured-link {
+  &:hover {
+    &::after {
+      background-color: $gray-light;
+      opacity: 1;
+      transition-duration: .3s;
+    }
+  }
+}
+
+// 3 featured logo / quote hover states
+.story-featured-cardLogoTop {
+  background-color: darken($rackspace-red, 5%);
+}
+
+.story-featured-cardLogoCenter {
+  background-color: $teal-dark;
+}
+
+.story-featured-cardLogoBottom {
+  background-color: $purple;
+}
+
+.story-featured-seeMore {
+  color: transparentize($black, .5);
+  display: block;
+  font-weight: $font-weight-black;
+  margin-top: 50px;
+  text-align: center;
+
+  &::before {
+    content: 'See More Customer Stories';
+    display: block;
+    font-family: $heading;
+    font-size: 20px;
+  }
+
+  &::after {
+    content: '\f0d7';
+    display: block;
+    font-family: 'FontAwesome';
```

Comment:
Also, it looks like its picking up a `:visited` and remaining bold after being clicked.